docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,39 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.
```

Comment:
add a sentence that gives a brief idea of what a subproject is, (a way of grouping / categorizing study participants by either experimental or diagnostic groups)

Reply:
@driusan before accepting the changes do you agree with the proposed description ?

Reply:
I dunno, I'd describe subprojects as "Ask 3 people what a subproject is if you want 4 answers".

The only way that the code uses it is for determining test battery. The sentence below is already misleading, so the proposed description doesn't make it any worse.

Reply:
"**Subprojects**, decided by the study as the experimental or diagnostic groups by which candidates are categorized, plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint."

Thoughts?

I think since "candidate" is use more frequently throughout LORIS, we should avoid inserting "study participant" in place of it where possible.

Reply:
@leighevans suprojects are not used by all projects for diagnostic groups. I'm gonna stick to simply saying they define the test battery because effectively thats the only thing they do